docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,320 @@
+---
+title: "Project"
+sidebar_position: 5
+description: >
+  `Project` is **a set of [Scope](../Glossary.md#data-scope) from different domains**, a way to group different resources, and it is crucial for some metric calculations like `Dora`.
+---
+
+# Summary
+
+In related calculations such as `Dora` indicators, we often encounter situations requiring comprehensive calculations based on data from multiple data sources.
+
+For example, we do code version management on `Gitlab`, and then do corresponding compilation and deployment operations in `Jenkins`. At this time, our indicator calculation will depend on the data of `Gitlab` and `Jenkins` at the same time.
+
+However, the reality is that sometimes, we have plural `Gitlab` connections and plural `Jenkins` connections.
+
+At this time, to support calculations that rely on multiple data sources like `Dora`, we need to correctly associate different [Scope](../Glossary.md#data-scope) under different data sources through a method.
+
+So, we created `Project`
+
+`Project` is **a set of [Scope](../Glossary.md#data-scope) from different domains**, a way to group different resources, and it is crucial for some metric calculation like `Dora`.
+
+By adding the `project_name` field to the `blueprint` table, we associate `project` with `blueprint` one-to-one.
```

Comment:
change "one-to-one" to "in one-to-one manner"
this sentence is describe the relationship between `project` and `blueprint`, which should be moved to the Models section.